messybrainz/db/recording.py

```diff
@@ -65,7 +65,7 @@ def fetch_distinct_recording_mbids(connection):
         SELECT DISTINCT rj.data ->> 'recording_mbid'
                    FROM recording_json AS rj
               LEFT JOIN recording_cluster AS rc
-                     ON (rj.data ->> 'recording_mbid')::uuid = rc.recording_gid
+                     ON (rj.data ->> 'recording_mbid') = rc.recording_gid::text
```

Comment:
Why this change? If PG can compare UUIDs, I would expect that to be faster than converting the UUID to text and then comparing it.

Reply:
thats for the speed. if i do it other way i.e. data ->>'recording_mbids'::uuid === raj.recording_mbid it gets slow